test_unstructured/partition/docx/test_docx.py

```diff
@@ -764,6 +764,31 @@ def test_partition_docx_includes_hyperlink_metadata():
     assert metadata.link_urls is None
 
 
+# -- shape behaviors -----------------------------------------------------------------------------
+
+
+def test_include_inline_shapes_text():
+    elements = partition_docx(example_doc_path("docx-inline-shapes.docx"))
```

Comment:
@ds-filipknefel Okay, now that I see the document I can better see what we're trying to do. So this is like a PowerPoint shape inserted in the text, not an image (PNG, JPG) per se. So that's not going to be complicated. Each shape will have (effectively) a text frame that can contain text, so there's no "text before the image", "text after the image" etc. (because there is no image).

So I think the test cases we have are fine. There just isn't more possible variation that we need to consider.